client/zedcloudapi_client.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/go-openapi/runtime"
 	httptransport "github.com/go-openapi/runtime/client"
 	"github.com/go-openapi/strfmt"
+	"github.com/hashicorp/go-retryablehttp"
```

Comment:
It says the code was generated. If so, do we regenerate it periodically? 'cause the change will be reverted during regeneraion procedure. If not, then you need to remove that `DO NOT EDIT` comment on the first line.

Reply:
I removed the comment. It is initially generated but there is no automated or manual re-generation intended. (Long-term, it would be ideal to be able to generate a client with a clean API from the swagger specs. But unfortunately the naming is very inconsistent and it needs many manual improvements of the generated code to make it compile. Out of scope here, we want to provide a Go API client open-source in the near future, which will then also replace this one. So this topic will come up for discussion soon.)